compatibility/compatibility-tests/build.gradle.kts

```diff
@@ -51,10 +51,7 @@ tasks.withType<Test>().configureEach {
   systemProperty("junit.jupiter.extensions.autodetection.enabled", "true")
 }
 
-// Compatibility tests fail on macOS with the following message: `libc++abi: terminating
-// with uncaught exception of type std::__1::system_error: mutex lock failed: Invalid argument`
-//
 // Compatibility tests fail, because Windows not supported by testcontainers (logged message)
-if ((Os.isFamily(Os.FAMILY_MAC) || Os.isFamily(Os.FAMILY_WINDOWS)) && System.getenv("CI") != null) {
```

Comment:
This will have the side-effect of enabling compatibility tests in CI. If they fail for whatever reason, I will re-disable them in CI. As long as I can run them locally it's fine :-)